test_double_linked_list.py

```diff
@@ -0,0 +1,85 @@
+from double_linked_list import DoubleLinked
+import pytest
+
+
+def test_insert_one():
+    linked = DoubleLinked()
```

Comment:
Use your docstrings! Even though you have descriptive test names, you shoul dbe in the habit of writing a good docstring for when you need more information than a function name can provide.